frontend/src/screens/settings/Backup.tsx

```diff
@@ -0,0 +1,460 @@
+                                <div>
+                                  <p>
+                                    As part of enabling VSS your hub will be
+                                    shut down, and you will need to enter your
+                                    unlock password to start it again.
+                                  </p>
+                                  <p className="mt-2">
+                                    Please ensure you have no pending payments
+                                    or channel closures before continuing.
+                                  </p>
+                                </div>
+                              </AlertDialogDescription>
+                            </AlertDialogHeader>
+                            <AlertDialogFooter>
+                              <AlertDialogCancel>Cancel</AlertDialogCancel>
+                              <AlertDialogAction
+                                onClick={() => migrateLDKStorage("VSS")}
+                              >
+                                Confirm
+                              </AlertDialogAction>
+                            </AlertDialogFooter>
+                          </AlertDialogContent>
+                        </AlertDialog>
+                      )}
+                    </div>
+                  </>
+                )}
+              </>
+            ) : (
+              <div className="flex flex-col gap-8">
+                <div>
+                  <p className="text-muted-foreground text-sm mb-4">
+                    Link your Alby Account to enable automated channel backups
+                    after each channels creation.
+                  </p>
+                  <Button
+                    type="button"
+                    variant={"secondary"}
+                    className="flex gap-2 justify-center"
+                    onClick={() => navigate("/alby/account")}
+                  >
+                    <Link2Icon className="w-4 h-4 mr-2" />
+                    Link Alby Account to Enable
+                  </Button>
+                </div>
+                <div className="flex flex-col gap-4">
+                  <div className="flex gap-2 mb-1 items-center">
+                    <h3 className="text-sm font-medium">
+                      Manual Channels Backup
+                    </h3>
+                    <Badge variant={"positive"}>Active</Badge>
+                  </div>
+                  <p className="text-muted-foreground text-sm ">
+                    To backup your channels state manually, without Alby Account
+                    linked, follow these steps:
+                  </p>
+                  <ol className="text-sm list-decimal list-inside">
+                    <li className="mb-1">
+                      Go to the working directory of your Alby Hub
+                    </li>
+                    <li>
+                      Back up the newest file in: Set your Unlock Passcode{" "}
```

Comment:
Did you test this? 
![image](https://github.com/user-attachments/assets/97b21156-6ee1-49a1-be53-93578e70cd39)

Reply:
resolved removed instructions and linked to guides